.github/workflows/ci.yml

```diff
@@ -0,0 +1,85 @@
+name: CI
```

Comment:
We'll want to delete this file; the 'main' workflow from calendra should be sufficient.

Reply:
Cool. Want to do the honours, or shall I?

Reply:
Done